### DISCUSSION TITLE
Pdflatex - changing the citation style

### DISCUSSION THREAD

**Discussion by rsbyrne** [executablebooks/meta#363] [Q&A]
Hi there!

I'm making great progress using Jupyter Book to prepare my PhD thesis - thanks to plenty of help from many of you.

I've been using the pdflatex service and it's been really awesome for the most part. One thing that's sticking at the moment is the citations. In the browser version, my citations (courtesy of the 'sphinx' section of the _config.yml) are in a neat, academic-looking author-year format:
![image](https://user-images.githubusercontent.com/11673079/118388351-44f92f00-b667-11eb-8217-2e7b1eefb0ab.png)

But when I go to produce the pdf, the citation styles revert to 'label', which will upset my examiners:
![image](https://user-images.githubusercontent.com/11673079/118388332-2004bc00-b667-11eb-9040-3c7580af502e.png)

Does anyone know off the top of their head how to force the LaTeX engine to use the author-year citations?

I imagine the answer to this question will also begin to answer some other questions I will certainly have before long, as there will be a number of unavoidable customisations I will have to make to get the university to accept my submission. (Let alone academic journals, which all have extremely specific requirements...)

I'm going to tag @mmcky only because I know you have some special insight into the pdflatex system. Sorry to pester!

Cheers,
Rohan

**Comment by mmcky** [comment]
hey @rsbyrne will need to do some testing around this but it is certainly a great feature if it isn't passing through at the moment (cc: @AakashGfude)
